traffic_ops/traffic_ops_golang/vendor/github.com/miekg/dns/edns.go

```diff
@@ -490,7 +490,7 @@ func (e *EDNS0_N3U) String() string {
 	return s
 }
 
-// EDNS0_EXPIRE implementes the EDNS0 option as described in RFC 7314.
+// EDNS0_EXPIRE implements the EDNS0 option as described in RFC 7314.
```

Comment:
Should we be editing vendored dependencies?

Reply:
Probably not, @jranson can you revert this one?

Reply:
definitely not..   that should be changed in the upstream dependency

Reply:
Thanks for catching, as I missed all of those deep vendor directories. They are reverted back. I'll submit PRs to the upstream projects for those typos.